TRB_label_assist/performance_eval.py

```diff
@@ -171,7 +172,10 @@ def cross_val_predict(model,
 
         # train the model
         logging.info("About to fit the model %s" % model)
-        model_.fit(train_trips)
+        if model_.__class__.__name__ == 'ClusterExtrapolationClassifier' :
```

Comment:
`ClusterExtrapolationClassifier` has dependence on `RefactoredNaiveCluster` ( as mentioned in comments  https://github.com/e-mission/e-mission-eval-private-data/pull/37/commits/e2448c5f584e483e089077062b0c74a2e178bf7e#r1302423250 ). So, the `Entry` type data is passed from `ClusterExtrapolationClassifier` to `RefactoredNaiveCluster` . if-else used because there are other model's`fit` functions that have no requirement of `Entry` data.

Reply:
Please see my comment above wrt unified interfaces.